pkgs/applications/editors/jetbrains/default.nix

```diff
@@ -29,36 +29,48 @@ let
         (
           cd $out/clion-${version}
           # bundled cmake does not find libc
-          rm -rf bin/cmake
-          ln -s ${cmake} bin/cmake
+          rm -rf bin/cmake/linux
+          ln -s ${cmake} bin/cmake/linux
```

Comment:
Looks good to. I wonder if we should switch to `autoPatchelfHook` to avoid breakages in future updates.

Reply:
Example: https://github.com/NixOS/nixpkgs/commit/ea5787ad5291ee1c131326cb9c9fec03d359edff